### DISCUSSION TITLE
File transfer

### DISCUSSION THREAD

**Discussion by 123Satheesh** [COVESA/dlt-daemon#589] [Q&A]
How to do file transfer from the target to local PC?
For example, my log DLT log file is available in the Apertis pro, if I want to transfer that file to local PC, how can I possible to transfer the file.

**Comment by minminlittleshrimp** [comment]
This might so far related to DLT viewer, maybe we can ask for help from DLT viewer expert to use plugin file transfer there.
Last time I visit, seems file transfer feature has some problems.
